index.php

```diff
@@ -11,6 +11,21 @@
  */
 define('IITPCONNECT_MINIMUM_PHP', '7.0');
 
+
+$fp = fileperms(dirname(__DIR__));
+$fpi =  (int)substr(sprintf('%o', $fp), -4);
```

Comment:
Is this check for all files and folders? We need to check all files and folders.